crates/fsrt/src/main.rs

```diff
@@ -175,6 +176,19 @@ fn scan_directory(dir: PathBuf, function: Option<&str>, opts: Opts) -> Result<Fo
     let name = manifest.app.name.unwrap_or_default();
 
     let paths = collect_sourcefiles(dir.join("src/")).collect::<HashSet<_>>();
+
+    let transpiled_async = paths
+        .iter()
+        .map(|path| {
+            if let Ok(data) = fs::read_to_string(path) {
+                if let Some(line) = data.lines().nth(0) {
+                    return line.contains("\"use strict\";");
```

Comment:
You might want to check for `'use strict'` as well, i.e. if they use single quotes. Also, you can use `https://doc.rust-lang.org/stable/std/option/enum.Option.html#method.is_some_and`, but you'll have to bump the docker image to use rustc 1.70 or higher.